app/components/TaskList.tsx

```diff
@@ -88,10 +122,17 @@ const TaskList: React.FC<Props> = ({ projectRevision }) => {
           color: blue;
         }
 
+        div :global(.highlight) {
+          background-color: #fafafc;
+        }
+        div :global(.bordered) {
+          border-bottom: 1px solid #d1d1d1;
+          padding: 10px 0 10px 0;
+        }
+
         div {
-          border: 2px solid black;
```

Comment:
I find it a bit weird to not have any border around the task list, @suhafa what do you think?

Reply:
Looks fine to me, the grey fill color acts like a block imo, so we don't really need to emphasize it with a border.